parsec/core/win_registry.py

```diff
@@ -211,3 +229,67 @@ def cleanup_parsec_drive_icons() -> None:
         _, pid = get_parsec_drive_icon(letter)
         if pid and not get_psutil().pid_exists(pid):
             del_parsec_drive_icon(letter)
+
+
+def add_parsec_mountpoint_directory_to_quick_access(
+    base_mountpoint_directory: pathlib.PurePath,
+    appguid: str | None = None,
+    appname: str | None = None,
+) -> None:
+    winreg = get_winreg()
+    appguid = APPGUID if appguid is None else appguid
+    appname = APPNAME if appname is None else appname
+    hkcu = winreg.HKEY_CURRENT_USER
+    base_key_1 = rf"Software\Classes\CLSID\{appguid}"
+    base_key_2 = rf"Software\Classes\Wow6432Node\CLSID\{appguid}"
+    systemroot = os.environ.get("SYSTEMROOT", rf"C:\Windows")
+    icon_path = _get_drive_icon_path()
+    for key in (base_key_1, base_key_2):
+        winreg.SetValue(hkcu, key, winreg.REG_SZ, appname)
+        winreg_write_user_dword(key, "SortOrderIndex", 0x42)
+        winreg_write_user_dword(key, "System.IsPinnedToNamespaceTree", 0x01)
+
+        winreg.SetValue(hkcu, rf"{key}\DefaultIcon", winreg.REG_SZ, rf"{icon_path},0")
+
+        winreg.SetValue(
+            hkcu, rf"{key}\InProcServer32", winreg.REG_SZ, rf"{systemroot}\system32\shell32.dll"
+        )
+
+        winreg.SetValue(hkcu, rf"{key}\Instance", winreg.REG_SZ, rf"")
+        winreg_write_user_string(
+            rf"{key}\Instance", "CLSID", "{0E5AAE11-A475-4c5b-AB00-C66DE400274E}"
+        )
+
+        winreg.SetValue(hkcu, rf"{key}\Instance\InitPropertyBag", winreg.REG_SZ, rf"")
+        winreg_write_user_dword(rf"{key}\Instance\InitPropertyBag", "Attributes", 0x11)
+        winreg_write_user_string(
+            rf"{key}\Instance\InitPropertyBag", "TargetFolderPath", rf"{base_mountpoint_directory}"
+        )
+
+        winreg.SetValue(hkcu, rf"{key}\ShellFolder", winreg.REG_SZ, rf"")
+        winreg_write_user_dword(rf"{key}\ShellFolder", "Attributes", 0xF080004D)
+        winreg_write_user_dword(rf"{key}\ShellFolder", "FolderValueFlags", 0x28)
+
+    key = rf"Software\Microsoft\Windows\CurrentVersion\Explorer\Desktop\NameSpace\{appguid}"
+    winreg.SetValue(hkcu, key, winreg.REG_SZ, appname)
+
+    key = rf"Software\Microsoft\Windows\CurrentVersion\Explorer\HideDesktopIcons\NewStartPanel"
+    winreg_write_user_dword(key, rf"{appguid}", 0x1)
+
+
+def remove_parsec_mountpoint_directory_to_quick_access(appguid: str | None = None) -> None:
```

Comment:
Should be `remove_parsec_mountpoint_directory_from_quick_access`

Reply:
done